### DISCUSSION TITLE
Export MaterialX made in LookdevX as USD in Maya 2025

### DISCUSSION THREAD

**Discussion by sharktacos** [Autodesk/maya-usd#3718] [Q&A] [answered]
Trying to export Maya geo with MaterialX shaders to USD in Maya 2025 using the new functionality in LookdevX to assign MaterialX shaders to native Maya geo. Not having any success, as the MaterialX shaders are not exporting with the USD. Is this currently supported in the USD export?

**Comment by Satyan1990** [comment, marked as answer]
Hi @sharktacos 
The workflow you are trying is not supported yet. After assigning MaterialX shaders to native Maya geo, we need to export the document as a Mtlx file and import it as a USD reference in a USD stage, then you will be able to assign the referenced material to USD assets.

Please note you might experience some issues with this workflow for now. For eg., When authoring the Mtlx document, all the nodes except the shader and output node should be grouped before exporting, otherwise, it will come in broken in a USD stage.

**Reply by sharktacos** [reply]
Awesome, just tested the workflow you described on a simple scene, and it is working. Everything displays nicely in VP2 and in the Arnold Render. I'm going to go play around with this and kick the tires! Thanks very much for the help!
